framework/src/media/audio/audio_manager.c

```diff
@@ -100,19 +103,22 @@ static int g_actual_audio_out_card_id = -1;
 static int get_avail_audio_in_card_id(void);
 static int get_avail_audio_out_card_id(void);
 static audio_manager_result_t get_active_audio_device_pcm(struct pcm **pcm, audio_device_type_t type);
+static audio_manager_result_t set_audio_volume(audio_device_type_t type, audio_config_t *audio_config, uint8_t volume);
 
 /****************************************************************************
  * Public Functions
  ****************************************************************************/
 audio_manager_result_t find_input_audio_card(void)
 {
+	int ret;
 	unsigned int card_id = -1;
 	unsigned int device_id = -1;
 	char type;
 	struct dirent *dir_entry;
 	DIR *dir_info;
+	char path[256] = "/dev/audio/";
```

Comment:
Audio using below,
AUDIO_MAX_DEVICE_PATH && CONFIG_AUDIO_DEV_PATH
CONFIG_AUDIO_DEV_PATH defined in audio/KConfig but AUDIO_MAX_DEVICE_PATH is not defined.

Why don't you use these values?
1. add CONFIG_AUDIO_MAX_DEVICE_PATH to KConfig
2. add defconfig CONFIG_AUDIO_MAX_DEVICE_PATH & CONFIG_AUDIO_DEV_PATH
3. clean audio.c & here

Reply:
Also we can use card_name in audio_card_info_s.
typedef struct audio_card_info_s audio_dev_info_t; -> this should be changed to audio_card_info_t

Reply:
- Okay. Let me add the config in another PR. Because this commit is already big enough..
- "audio_card_into_t" is now used.